cfglogging.go

```diff
@@ -12,7 +13,14 @@ func cfgLogging(cmd *CmdCtx, cCtx *cli.Context) error {
 	if logLevel == "" {
 		logLevel = "INFO"
 	}
-	logger.New(logLevel, logger.WithConsole())
-	cmd.log = logger.Sugar
+	if logLevel == "TEST" {
```

Comment:
In the service level we have `NOOP` for tests that call the logger. 

Would it be good to be consistent with that? or maybe a more descriptive `OFF`, so it can be used in different scenarios other than testing

Reply:
This has to be TEST. Because veracity uses NOOP to cleanly configure output via logging. The TEST channel is actually consistent with how the loggig package works. OFF is not a channel it supports